CodeGen/include/Luau/IrData.h

```diff
@@ -185,6 +185,11 @@ enum class IrCmd : uint8_t
     // A: double
     SIGN_NUM,
 
+    // Select B if C == D, otherwise select A
+    // A, B: double (endpoints)
+    // C, D: double (condition arguments)
+    SELECT_NUM,
```

Comment:
E is not documented (even if only 1 value is supported).
Noticed that E is still provided in `translateBuiltinMathLerp`

Reply:
Ah, I forgot to remove it - initially was going to support it but it ended up too much code that’s not currently useful. I’ll remove extra argument from lerp

Reply:
Removed